tests/test_tags.py

```diff
@@ -93,18 +92,3 @@ def test_tag_mixed_abi(monkeypatch):
         'platlib': [f'extension{ABI3SUFFIX}', f'another{SUFFIX}'],
     })
     assert str(builder.tag) == f'{INTERPRETER}-{ABI}-{PLATFORM}'
-
-
-@pytest.mark.skipif(platform.system() != 'Darwin', reason='macOS specific test')
-def test_tag_macos_build_target(monkeypatch):
-    monkeypatch.setenv('MACOS_BUILD_TARGET', '12.0')
```

Comment:
I see here that is touched

Reply:
Sure, but who reads it? Which piece of software reads this environment variable? The environment variable expected by the toolchain is `MACOSX_DEPLOYMENT_TARGET`. I suspect that `MACOS_BUILD_TARGET` is something I made up.

Reply:
Yes. but when this test will set variable to 10.9 and the tag calculation will check files then this test will fail, as the test will in both runs return 12.0 (as the wrong variable was set).

Reply:
@Czaki you don't make any sense. I'm giving up.

Reply:
Your assumption that after correctly setting the environment variable, the toolchain will produce the correct files is wrong. It will often be true, but not always. 

Nad I only point the test that, if checking the platform tag will examine files, the test will find a bug in the variable name.

Reply:
Except that the test is not compiling any code an there is no file to examine.

Reply:
After reading the code I was pretty sure that it compile. I'm sorry for bothering you.